druid/src/widget/container.rs

```diff
@@ -135,29 +135,35 @@ impl<T: Data> Widget<T> for Container<T> {
     }
 
     fn paint(&mut self, paint_ctx: &mut PaintCtx, data: &T, env: &Env) {
-        let panel = RoundedRect::from_origin_size(
-            Point::ORIGIN,
-            paint_ctx.size().to_vec2(),
-            self.corner_radius,
-        );
-
-        if let Err(e) = paint_ctx.save() {
-            log::error!("{}", e);
-            return;
-        }
-
-        paint_ctx.clip(panel);
-
         if let Some(background) = self.background.as_mut() {
+            let bg_rect = RoundedRect::from_origin_size(
```

Comment:
my current preferred way to do this (until https://github.com/linebender/kurbo/pull/92 merges) is `paint_ctx.to_rect().to_rounded_rect(self.corner_radius);` 😒

Reply:
Using the helper methods now!